google_auth_oauthlib/tool/__main__.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (C) 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Command-line tool for obtaining authorization and credentials from a user.
+
+This tool uses the OAuth 2.0 Authorization Code grant as described in
+`section 1.3.1 of RFC 6749`__ and implemeted by
+:class:`google_auth_oauthlib.flow.Flow`.
+
+This tool is intended to bootstrap development in situation when the
+application cannot easily run the 3LO OAuth2 flow, i.e: in an embedded
+device with limited input / display capabilities.
+
+This is not intended for production where the main application should
+run the 3LO OAuth2 flow to get authorization from the users.
+
+..  __: https://tools.ietf.org/html/rfc6749#section-1.3.1
```

Comment:
Nit: please use rst named references:

```python
.. _section 1.3.1 of RFC 6749: https://tools.ietf.org/html/rfc6749#section-1.3.1
```

Just in case we add more links.